### ISSUE TITLE
Uncaught TypeError: Cannot read property 'dispose' of undefined

### ISSUE THREAD

**Issue by kieranklaassen** [abe33/minimap-pigments#1]
1. Update to minimap 4.13.0 in the settings pane

**Atom Version**: 1.0.7
**System**: Mac OS X 10.10.5
**Thrown From**: [minimap-pigments](https://github.com/abe33/minimap-pigments) package, v0.1.5
### Stack Trace

Uncaught TypeError: Cannot read property 'dispose' of undefined

```
At /Users/kieranklaassen/.atom/packages/minimap-pigments/lib/minimap-pigments.coffee:45

TypeError: Cannot read property 'dispose' of undefined
  at /Users/kieranklaassen/.atom/packages/minimap-pigments/lib/minimap-pigments.coffee:45:38
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:273:27)
  at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app.asar/src/model.js:45:58)
  at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:460:18)
  at Pane.module.exports.Pane.destroyActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:435:12)
  at Workspace.module.exports.Workspace.destroyActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:617:35)
  at Workspace.module.exports.Workspace.destroyActivePaneItemOrEmptyPane (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:656:21)
  at atom-workspace.atom.commands.add.core:close (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace-element.js:290:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```
### Commands

```
     -1:07.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:01.8.0 core:close (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "railscast-theme"
    ],
    "disabledPackages": [
      "language-coffee-script-angular"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
aligner-ruby, v1.4.0
aligner-scss, v1.0.2
atom-beautify, v0.28.11
color-picker, v2.0.11
coverage, v0.6.1
file-icons, v1.6.6
html2haml, v0.10.0
language-haml, v0.21.0
linter, v1.4.3
linter-coffeelint, v1.0.2
linter-haml, v1.0.0
linter-reek, v2.0.0
linter-rubocop, v0.3.2
linter-ruby, v1.0.2
linter-scss-lint, v2.0.0
minimap, v4.13.0
minimap-color-highlight, v4.1.4
minimap-find-and-replace, v4.3.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.0
minimap-pigments, v0.1.5
pigments, v0.11.0
railscast-theme, v0.3.0
wercker-status, v1.0.5

# Dev
No dev packages
```

**Comment by abe33** [comment]
Thanks for the report, looks like I forgot something, will push a patch soon.

**Comment by kieranklaassen** [comment]
Awesome! Thanks for the plugin!

**Comment by abe33** [comment]
Fixed in v0.1.6.
